CODE/Sources/PennySHARED/Repositories/DynamoUserRepository.swift

```diff
@@ -3,47 +3,37 @@ import SotoDynamoDB
 import PennyModels
 import PennyExtensions
 
-public enum DBError: Error {
-    case invalidItem
-    case tableNameNotFound
-    case invalidRequest
-    case invalidHandler
-    case itemNotFound
-}
-
-public struct DynamoUserRepository: UserRepository {
+struct DynamoUserRepository: UserRepository {
     
     // MARK: - Properties
     let db: DynamoDB
     let tableName: String
-    let eventLoop: EventLoop
```

Comment:
We should bring this back - the event loop from the DynamoDB instance isn't necessarily the same event loop as the context it's being called from. It's probably fine for now but making it explicit ensures we don't shoot ourselves in the foot in the future